src/solidity-frontend/solidity_convert.cpp

```diff
@@ -2057,11 +2058,17 @@ bool solidity_convertert::get_decl_ref_builtin(
   // -ve ref id means built-in functions or variables.
   // Add more special function names here
   assert(
-    decl["name"] == "assert" || decl["name"] == "__ESBMC_assume" ||
-    decl["name"] == "__VERIFIER_assume");
+    (decl["name"] == "assert" || decl["name"] == "require" ||
```

Comment:
Shall we also add `assume` here?

Reply:
I don't think so. Because there is no "assume" keyword in Solidity -- they use "require" instead, thus we have `decl["name"] == "require"`